src/main/java/com/openlattice/postgres/PostgresTable.java

```diff
@@ -288,7 +289,11 @@
                     .addColumns( ID, NULLABLE_TITLE, DESCRIPTION, ALLOWED_EMAIL_DOMAINS, MEMBERS, APP_IDS );
     public static final PostgresTableDefinition        PERMISSIONS                  =
             new PostgresTableDefinition( "permissions" )
-                    .addColumns( ACL_KEY, PRINCIPAL_TYPE, PRINCIPAL_ID, PostgresColumn.PERMISSIONS )
+                    .addColumns( ACL_KEY,
```

Comment:
You'll want a migration in mechanic that executes the necessary alter table statement.

Reply:
I believe I have a PR in mechanic for this